hugo.toml

```
@@ -0,0 +1,69 @@
+baseURL = 'https://tina.pm/'
+languageCode = 'en-us'
+title = "Tina's team website"
+theme = 'introduction'
+
+# For a configuration example of introduction theme, see
+# https://github.com/victoriadrake/hugo-theme-introduction/blob/master/exampleSite/config.toml
+
+DefaultContentLanguage           = "en"                    # Default language for multilingual sites
+
+[params]
+  themeStyle                   = "auto"                  # Choose "light" or "dark" or "auto"
+  # favicon                      = "/img/fav.ico"          # Path to favicon file
+  showMenu                     = true                    # Show navigation menu
+  showRSSButton                = false                   # Show rss button in navigation
+  fadeIn                       = true                    # Turn on/off the fade-in effect
+  fadeInIndex                  = false                   # Turn on/off the fade-in effect on the index page even if fade-in was otherwise turned off
+  dateFormat                   = "Jan 2, 2006"
+  email                        = "info@tina.pm"          # E-mail address for contact section
+
+  # Configure the home page
+  [params.home]
+      introHeight              = "fullheight"            # Input either "medium" or "large" or "fullheight"
+      localTime                = false                   # Show your current local time in contact section
+      timeZone                 = "Europe/Dublin"         # Your timezone as in the TZ* column of this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+      timeFormat               = "en-US"                 # Language specific format to use
```

similarly, this will produce a 12-hour format, while Europe mostly uses 24h
```suggestion
      timeFormat               = "en-GB"                 # Language specific format to use
```